### PR TITLE
Resolves #1841

### LINKED ISSUE #1841: lDAPDisplayNames Should not Include the String ms-DS
The names of the attributes for the default domain policy do not include the string "ms-DS". Only Fine-Grained Password Policy objects have attribute names with this string.

Description of LockoutDuration parameter states the lDAPDisplayName is "msDS-LockoutDuration". It should be "lockoutDuration".

Description of LockoutObservationWindow parameter states the lDAPDisplayName is "msDS-lockoutObservationWindow". It should be "lockoutObservationWindow".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: da60451d-27e5-c1d1-9a8d-a25adab2c1bb
* Version Independent ID: f4d36700-18b7-9ead-fb31-da0c5e912c1b
* Content: [Set-ADDefaultDomainPasswordPolicy (activedirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/set-addefaultdomainpasswordpolicy?view=winserver2012-ps#feedback)
* Content Source: [docset/winserver2012-ps/activedirectory/Set-ADDefaultDomainPasswordPolicy.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/winserver2012-ps/activedirectory/Set-ADDefaultDomainPasswordPolicy.md)
* Product: **w10**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix LDAP display names.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1841

### CODE REVIEW

**Comment by e0i** [comment]
cc @dariomws @get-itips @tseward

File: docset/windows/addsadministration/Set-ADDefaultDomainPasswordPolicy.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ Accept wildcard characters: False
 Specifies the length of time that an account is locked after the number of failed login attempts exceeds the lockout threshold.
 You cannot login to an account that is locked until the lockout duration time period has expired.
 This parameter sets the **lockoutDuration** property of a password policy object.
-The LDAP display name (**ldapDisplayName**) of this property is msDS-LockoutDuration.
```
Comment (tseward) [inline docset/windows/addsadministration/Set-ADDefaultDomainPasswordPolicy.md:190]: It is likely better to leave the "msDS-" as originally presented given it is the name of the attribute and is easily searchable if you want to know more information about it.

Comment (velkovb) [reply docset/windows/addsadministration/Set-ADDefaultDomainPasswordPolicy.md:190]: https://docs.microsoft.com/windows/win32/adschema/a-lockoutduration
https://docs.microsoft.com/windows/win32/adschema/a-msds-lockoutduration

This is confusing... Which one is the real one?

Comment (get-itips) [reply docset/windows/addsadministration/Set-ADDefaultDomainPasswordPolicy.md:190]: It seems _lockoutduration_ and _ms-ds-lockoutduration_ belong to different classes.
_ms-ds-lockoutduration_ belongs to _ms-DS-Password-Settings_ the schema articles that you shared confirms that, and it is to be found in the Password Settings Container of a W2008 Functional Level.

What we need to know is to confirm which one is affected by this cmdlet, can you run this cmdlet in your lab to check ?

**Comment by velkovb** [comment]
image
This affects the defaults ones. I would suppose the others are related to fine grained password policies.

**Comment by get-itips** [comment]
@velkovb  Yes, what I thought, so I think we should use LockoutDuration if that is the affected attribute.

**Comment by velkovb** [comment]
@tseward Could you review again?

**Comment by tseward** [comment]
Yep, my mistake.

image

**Pull request merged**